tests/suites/0_stateless/13_tpch/13_0005_q5.sql

```diff
@@ -15,6 +15,7 @@ where
   and l_suppkey = s_suppkey
   and c_nationkey = s_nationkey
   and s_nationkey = n_nationkey
+  and c_nationkey = n_nationkey
```

Comment:
May need to remove.

Reply:
The condition will be derived by most systems, we also need to derive it but currently, we don't support functional dependency, so I add it manually